src/lib/temple/back/vault/vault.test.ts

```diff
@@ -8,6 +8,8 @@ const mnemonic = 'street seminar popular skill actress route treat coral ready b
 const accountName = 'Temple';
 
 describe('Vault tests', () => {
+  jest.setTimeout(10000);
```

Comment:
what is this for?

Reply:
Unfortunately, I had to increase tests timeout to make them pass locally